CMakeLists.txt

```diff
@@ -448,13 +448,9 @@ include(SwiftList)
 swift_configure_components()
 
 # lipo is used to create universal binaries.
-if("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
-  include(SwiftDarwin)
-  if(SWIFT_LIPO)
-    set(LIPO ${SWIFT_LIPO})
-  else()
-    find_toolchain_tool(LIPO "${SWIFT_DARWIN_XCRUN_TOOLCHAIN}" lipo)
-  endif()
+include(SwiftDarwin)
+if(NOT SWIFT_LIPO)
+  find_toolchain_tool(SWIFT_LIPO "${SWIFT_DARWIN_XCRUN_TOOLCHAIN}" lipo)
```

Comment:
Please do not include SwiftDarwin unconditionally. How does this work on non-darwin platforms? Is SWIFT_LIPO set to something so we don't invoke xcrun?

Reply:
It doesn't really pollute anything, so I'm not sure what the harm is.  You can always set it to something on the command line to avoid running xcrun.